pom.xml

```diff
@@ -61,7 +62,13 @@
             <url>https://packages.confluent.io/maven/</url>
         </repository>
     </repositories>
-
+    <distributionManagement>
```

Comment:
Why do you need have it?
You can omit the distributionManagement part entirely in your poms and set it either on your build server or in settings.xml.